openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -35,16 +35,20 @@
         LDAP, Active Directory, Red Hat SSO, etc). This part of the control
         is not applicable to OpenShift.
 
-        To aid system administrators, documentation on
+        To aid system administrators in configuring a centralized identity provider, documentation on
         supported authentication providers can be found in the
         OpenShift product documentation:
 
-        https://access.redhat.com/documentation/en-us/openshift_container_platform/latest/html/authentication/understanding-identity-provider
+        https://docs.openshift.com/container-platform/latest/authentication/understanding-authentication.html
 
         The ability to uniquely identify processes acting on behalf of
         organizational users, such as when a user deploys a new
         container image, is provided by correlating system audit logs.
 
+        Currently, uniquely identifing containers in OpenShift cannot be done. This is being tracked in the following ticket:
+
+        https://issues.redhat.com/browse/CMP-299
+
```

Comment:
Let's remove this one for now. I'll pass you the actual link to where this is being tracked. The rest of the items look good to me.

Reply:
Removed. Thanks!